docs/conf/prague/2022/tickets.rst

```diff
@@ -100,6 +100,21 @@ Purchase this ticket if you are currently enrolled as a student, or don't curren
 
 .. class:: ticket
 
+**Corporate Concierge Tickets** *{{tickets.concierge.price}}*
+------------------------------------------------------
+
+We offer a corporate concierge service if your company is unable to follow our regular ticket sales process through the website.
+We can offer payment by invoice, process purchase orders, bank transfers, fill in supplier registration forms, and offer other support.
+Note that payment must still be received in full before the start of the conference.
```

Comment:
Should we add something extra-explicit like "we reserve the right to cancel registrations for unpaid tickets" or something like that? To make it clear that if they don't pay then they won't be able to log into the event.

Reply:
I think a reminder during the process will fit better. Once we try this, if it is actually used enough, I can come up with some templates in helpmonks.

Reply:
Yea, I don't think we need stronger language here. I'm also mostly fine allowing payment after the date, but it's a good policy to start with.